hack/make-rules/test.sh

```diff
@@ -312,7 +312,7 @@ for (( i=0; i<${apiVersionsCount}; i++ )); do
   apiVersion=${apiVersions[i]}
   echo "Running tests for APIVersion: $apiVersion"
   # KUBE_TEST_API sets the version of each group to be tested.
-  KUBE_TEST_API="${apiVersion}" runTests "$@"
+  KUBE_TEST_API="${apiVersion}" KUBE_API_VERSIONS="${apiVersion}" runTests "$@"
```

Comment:
My understanding is that later we will remove the for loop, and set `KUBE_API_VERSIONS` to equal to `apiVersions`. A test that needs a specific version will construct its own `APIRegistrationManager`. Am I right?